unic/char/property/src/macros.rs

```diff
@@ -26,7 +26,7 @@
 ///         human => "Human-Readable Property Name";
 ///
 ///         /// Exactly one attribute
```

Comment:
Update this to something like "any number of doc comments"; it's not accurate like this anymoe